.github/workflows/lint.yml

```diff
@@ -29,14 +29,14 @@ jobs:
         with:
           jdk-version: 21
 
-      - name: Lint project
-        run: |
-          ./gradlew \
-            detekt \
-            ktlintCheck \
-            vale
-        # TODO: migrate to the shared lint script
-        # run: ./mobile-android-pipelines/scripts/lintProject.sh "" ""
+      - name: Lint Kotlin with detekt
+        run: ./gradlew detekt
```

Comment:
Is the pipe symbol no longer needed here (before the `./gradle ...`) commands since it's just a single line command now?